### PR TITLE
Add pyrdl

### PR DESCRIPTION
**Pull request by mjohnson541**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mjohnson541** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by mjohnson541** [comment]
@conda-forge/staged-recipes ready for review!

**Comment by mjohnson541** [comment]
@conda-forge-admin, please ping team

File: recipes/pyrdl/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    CMAKE_FLAGS+=" -DPython_ROOT_DIR=${PREFIX}"
+    CMAKE_FLAGS+=" -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT}"
+    CMAKE_FLAGS+=" -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}"
+fi
+
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_PYTHON_WRAPPER=ON
+make
+
```
Comment (ocefpaf) [inline recipes/pyrdl/build.sh:15]: Is there a `make check` or `ctest` that we can run here?

File: recipes/pyrdl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% set name = "pyrdl" %}
+{% set version = "1.1.3_rdkit" %}
```
Comment (ocefpaf) [inline recipes/pyrdl/meta.yaml:2]: I'm assuming that the `_rdkit` is not part of the versioning scheme, right?
```suggestion
{% set version = "1.1.3" %}
```

File: recipes/pyrdl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/pyrdl/meta.yaml:9]:
```suggestion
  url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v{{ version }}_rdkit.tar.gz
```

File: recipes/pyrdl/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  host:
+    - python
+    - cython
+  run:
+    - python
+
+test:
+  source_files:
+    - 'build/*'
+
```
Comment (ocefpaf) [inline recipes/pyrdl/meta.yaml:33]:
```suggestion
```

File: recipes/pyrdl/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - py_rdl
```
Comment (ocefpaf) [inline recipes/pyrdl/meta.yaml:35]: Are you using the `build/*` files in the import test? If so this package will be broken b/c the test files are not package. If not you can drop the test files b/c I don't see other tests that are using them.

**Review by awvwgk — CHANGES_REQUESTED** [review]
Note that the ringdecomposerlib is already packaged on conda-forge: https://github.com/conda-forge/ringdecomposerlib-feedstock

**Comment by mjohnson541** [comment]
Hmm...in that case would it be better to just add the -DBUILD_PYTHON_WRAPPER=ON flag and import test to the ringdecomposerlib feedstock instead?

**Comment by awvwgk** [comment]
The Python wrappers link statically against ringdecomposerlib, this means the build files must be patched in any case to prefer dynamic linking. You can either have ringdecomposerlib as requirement in host for this recipe or we add an output to the existing feedstock producing the Python wrapper.

**Comment by awvwgk** [comment]
I'll build the Python wrapper as additional output in ringdecomposerlib (see https://github.com/conda-forge/ringdecomposerlib-feedstock/pull/2). Any preference on the naming? Either `ringdecomposerlib-python` or `py_rdl` seem like possible choices to me.

**Comment by mjohnson541** [comment]
Both of those sound good! Thank you so much!

**Pull request closed without merging**